frontend/src/ui/Blocks/Series.tsx

```diff
@@ -74,18 +84,48 @@ type FromSeriesProps = SharedFromSeriesProps & {
     fragRef: SeriesBlockSeriesData$key;
 };
 
-export const SeriesBlockFromSeries: React.FC<FromSeriesProps> = ({ fragRef, ...rest }) => {
+const SeriesBlockFromSeries: React.FC<FromSeriesProps> = (
+    { fragRef, ...rest },
+) => {
```

Comment:
Why was that split into three lines if the only other change made the line shorter? :thinking: 

And do we even need this component anymore? It's not used anymore, so why not remove that and use its name for the new component? I.e. `SeriesBlockFromSeries` expects a ready series.